Vic2ToHoI4/Source/Configuration.cpp

```diff
@@ -162,7 +162,11 @@ Configuration::Factory::Factory()
 			 std::clamp(static_cast<float>(commonItems::singleDouble{theStream}.getDouble()), 0.0F, 100.0F) / 100.0F;
 		Log(LogLevel::Info) << "\tPercent of commanders: " << configuration->percentOfCommanders;
 	});
-	registerRegex(commonItems::catchallRegex, commonItems::ignoreItem);
+	registerKeyword("output_name", [this](const std::string& unused, std::istream& theStream) {
+		const commonItems::singleString outputNameStr(theStream);
+		configuration->customOutputName = outputNameStr.getString();
+		registerRegex(commonItems::catchallRegex, commonItems::ignoreItem);
```

Comment:
You moved this regex inside this lambda by accident. Restore it back at the end.